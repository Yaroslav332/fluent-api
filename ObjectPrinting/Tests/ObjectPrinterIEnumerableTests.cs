using System;
using ApprovalTests;
using ApprovalTests.Reporters;
using NUnit.Framework;

namespace ObjectPrinting.Tests
{
    [UseReporter(typeof(DiffReporter))]
    [TestFixture]
    public class ObjectPrinterIEnumerableTests
    {
        [Test]
        public void Print_Should_CorrectlyDisplayIEnumerables()
        {
            var e = new Enumerics();
            var printer = ObjectPrinter.For<Enumerics>();
            Approvals.Verify(printer.PrintToString(e));
        }
    }
}