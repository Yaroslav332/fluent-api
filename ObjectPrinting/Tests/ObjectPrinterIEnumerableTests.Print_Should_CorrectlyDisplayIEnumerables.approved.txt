Enumerics
	A = [
		0,
		0,
		0,
		0,
		]
	L = [
		Alex,
		Vovan,
		Peter,
		]
	D = [
		KeyValuePair`2
			Key = Person
				Id = Guid
				Name = Anna
				Height = 0
				Age = 0
			Value = Operation
				Operator1 = 1
				Operator2 = 0
				Operator3 = 0
				Operator4 = 0
				Operator5 = 0
				Operator6 = 0
				OperationDate = 01.01.0001 0:00:00,
		KeyValuePair`2
			Key = Person
				Id = Guid
				Name = Winona
				Height = 0
				Age = 0
			Value = Operation
				Operator1 = 3
				Operator2 = 0
				Operator3 = 0
				Operator4 = 0
				Operator5 = 0
				Operator6 = 0
				OperationDate = 01.01.0001 0:00:00,
		]
